crates/ln-dlc-node/src/ln/common_handlers.rs

```diff
@@ -254,8 +254,16 @@ where
             }
         }
 
-        node.sub_channel_manager
-            .notify_ln_channel_closed(channel_id)?;
+        match node
+            .sub_channel_manager
+            .notify_ln_channel_closed(channel_id)
+        {
+            Ok(()) => {}
+            Err(dlc_manager::error::Error::InvalidParameters(msg)) => {
+                tracing::debug!("Irrelevant LDK closure notification: {msg}");
+            }
+            e @ Err(_) => e.context("Failed to notify subchannel manager about LDK closure")?,
```

Comment:
`e @ Err(_)` - what sort of wizardry is this? never seen it before! 😳

Reply:
And it's cooler than I thought! Look at the second example here: https://doc.rust-lang.org/rust-by-example/flow_control/match/binding.html.

Reply:
really nice. I will add it to my repertoire ;)